kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -63,48 +75,83 @@
       targetTime() {
         return this.totalPages * 30;
       },
+      documentLoading() {
+        return this.progress !== 1;
+      },
     },
     methods: {
       toggleFullscreen() {
-        if (this.isFullscreen) {
-          if (this.fullscreenAllowed) {
-            ScreenFull.toggle(this.$refs.container);
-          }
-          this.isFullscreen = false;
+        if (this.fullscreenAllowed) {
```

Comment:
A minor fullscreen bugfix. Might be best to break this into a separate PR :man_shrugging: